swagger_to/swagger.py

```diff
@@ -289,22 +306,34 @@ def parse_path(adict: MutableMapping[str, Any]) -> Tuple[Path, List[str]]:
     return pth, errors
 
 
-def parse_yaml(stream: Any) -> Tuple[Swagger, List[str]]:
+def parse_yaml(stream: Any, with_line_number: bool) -> Tuple[Swagger, List[str]]:
     """
     Parses the Swagger specification from the given text.
 
     :param stream: YAML representation of the Swagger spec satisfying file interface
+    :param with_line_number: if set, the swagger definition will contain line numbers at each node
```

Comment:
(Comment B) I would remove the parameter `with_line_number` from `parse_yaml` function and just always add the line numbers for simplicity.

Reply:
That's what I did in the beginning, but then line numbers would be included in the autogenerated jsonschemas.go in swagger_to_go_server.

Reply:
...Which fixes itself when moving __lineno__ out of the adict. I'll remove the flag.